data/base-map-and-imagery/index.md

```diff
@@ -35,26 +35,26 @@ Do you need a quick, fast base map for your [GIS, CAD, or web services]({{ "/dis
 {: .pop .text-center }
 
-AGRC’s cloud-based server, [Discover]({{ "/discover/" | prepend: site.baseurl }}), provides imagery and base map services in an OGC standard WMTS and WMS in the Web Mercator WGS84 projection (wkid: 3857). There are over [20 services]({{ "/discover/#services" | prepend: site.baseurl }}) available from Discover that are free to use. **A subset of services do require a [signed license agreement]({{ "/discover/license" | prepend: site.baseurl }})**.
+AGRC’s cloud-based server, [Discover]({{ "/discover/" | prepend: site.baseurl }}), provides imagery and base map services in an OGC standard WMTS and WMS in the Web Mercator WGS84 projection (wkid: 3857). Discover offers over [20 services]({{ "/discover/#services" | prepend: site.baseurl }}) that are free to use (**but a few services do require a [signed license agreement]({{ "/discover/license" | prepend: site.baseurl }})**.
 
-<i class="fas fa-fw fa-info-circle"></i> Each cache consists of around **3 to 17 million** 256 pixel square tiles that can range from **15G to over 200GB**. These caches are standardized on a subset of the Google, Microsoft, and esri [scale levels]({{ "/developer/base-maps/basemap-scales/" | prepend: site.baseurl }}) that apply to our state. The [update schedule]({{ "/developer/base-maps/basemap-maintenance-schedule/" | prepend: site.baseurl }}) will keep you in the loop as to when the tiles were last updated. If you would like to download imagery visit the [aerial photography]({{ "/data/aerial-photography/" | prepend: site.baseurl }}) data page.
+<i class="fas fa-fw fa-info-circle"></i> Each cache consists of around **3 to 17 million** 256-pixel square tiles ranging from **15G to over 200GB**. These caches are standardized on a subset of the Google, Microsoft, and esri [scale levels]({{ "/developer/base-maps/basemap-scales/" | prepend: site.baseurl }}) that apply to our state. The [update schedule]({{ "/developer/base-maps/basemap-maintenance-schedule/" | prepend: site.baseurl }}) will keep you in the loop as to when the tiles were last updated. If you would like to download imagery, visit the [aerial photography]({{ "/data/aerial-photography/" | prepend: site.baseurl }}) data page.
 
 #### The Services on Discover
 
-<i class="fas fa-fw fa-map"></i> **Terrain**: This base map highlights elevation with mountain peak elevation labels, contour lines, ski resorts, as well as many other places of interest.
+<i class="fas fa-fw fa-map"></i> **Terrain**: This base map highlights elevation and features mountain peak elevation labels, contour lines, and locations of ski resort locations and other places of interest.
 
 <i class="fas fa-fw fa-map"></i> AGRC maintains a very minimal cache called **Lite**. It is very muted in color and does not distract from data overlayed on top of it.
```

Comment:
Maybe a ridiculous question, but "overlayed": so the past tense of lay is "laid," so should this be "overlaid"? Or is "overlayed" industry standard?

Reply:
it should be overlaid